docs/src/main/tut/optiont.md

```diff
@@ -51,6 +51,33 @@ val noWelcome: OptionT[Future, String] = customGreetingT.filterNot(_.contains("w
 val withFallback: Future[String] = customGreetingT.getOrElse("hello, there!")
 ```
 
+## From `Option[A]` and/or `F[A]` to `OptionT[F, A]`
+
+Sometimes you may have an `Option[A]` and/or `F[A]` and want to *lift* them into an `OptionT[F, A]`. For this purpose `OptionT` exposes two useful methods, namely `fromOption` and `liftF`, respectively. E.g.:
+
+```tut:silent
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import cats.data.OptionT
+import cats.std.future._
```

Comment:
Yes, I noticed but I wanted to example to be *self-contained*. 

By the way, if you don't import `scala.concurrent.ExecutionContext.Implicits.global` you get a misleading: 
```scala
scala> OptionT.liftF(f)
<console>:17: error: could not find implicit value for parameter F: cats.Functor[scala.concurrent.Future]
       OptionT.liftF(f)
```
Do you think I should file this in a different issue or it's not a problem? I mean, even if you import `cats.std.future._` you get that error if no `ExecutionContext` is found in scope and people could get confused by this because they would think the problem is it cannot find a `Functor[Future]` instance while it actually can't just find the `ExecutionContext`. 

Anyway if you think it's not a problem I can remove those imports.